### PR TITLE
Fixes #416 - Updated account instructions for registering with keycloak

### LINKED ISSUE #416: Update IN-CORE Account instructions for registering with keycloak
The LDAP registration page and links in the documentation need to be updated to point to the keycloak signup.

### PR DESCRIPTION
**Pull request by navarroc**
The account registration instructions and FAQ are updated. Once we decide if we are enabling reCAPTCHA, I will need to update the image file.

### CODE REVIEW

**Review by longshuicy — APPROVED** [review]
Looks good. Could you resolve the merge conflict whenever you have time?

File: manual_jb/content/account.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-the system and allows you to create data that is only accessible by you. If applicable, it is recommended
-that you sign up with your institutional email. To register for an account, please go to:
+Access to the **IN-CORE** service requires a user account. Registering for an account is free and open to everyone.
+This account gives you access to all of the public data on the system and allows you to create data that is only
+accessible by you. If applicable, it is recommended that you sign up with your institutional email.
+To register for an account, please go to:
 
-[https://identity.ncsa.illinois.edu/register/BSKC2UKQPU](https://identity.ncsa.illinois.edu/register/BSKC2UKQPU)
+[https://incore.ncsa.illinois.edu/auth/realms/In-core/login-actions/registration?
+client_id=react-auth](https://incore.ncsa.illinois.edu/auth/realms/In-core/login-actions/registration?client_id=react-auth)
```
Comment (longshuicy) [inline manual_jb/content/account.md:9]: You might need to update this to another pattern:
https://incore.ncsa.illinois.edu/auth/realms/In-core/protocol/openid-connect/registrations?client_id=react-auth&response_type=code&scope=openid&redirect_uri=https://incore.ncsa.illinois.edu/

File: manual_jb/content/faq.md [hunk 2]
```diff
@@ -4,13 +4,11 @@
 
 *   *I want to try your IN-CORE services. Do I need to register?*
 
-    A user must have an account [registered](https://identity.ncsa.illinois.edu/register/BSKC2UKQPU) with NCSA IN-CORE service. User credentials are required 
-    in accessing repositories such as hazard, fragility, restoration, geographic and other data sets. 
-    They are also used for accessing documentation server and Jupyter Notebook files.
+    A user must have an account [registered](https://incore.ncsa.illinois.edu/auth/realms/In-core/login-actions/registration?client_id=react-auth`) with NCSA's IN-CORE service.
```
Comment (longshuicy) [inline manual_jb/content/faq.md:7]: Same you might need to update the registration link

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: manual_jb/content/account.md (hunk 1 above)
Comment (navarroc) [reply manual_jb/content/account.md:9]: Good catch! I will update this shortly.

**Comment by navarroc** [comment]
I have fixed the problem @longshuicy found; however, to test it, you will need make one small change and build the docs locally. The link is very specific to the instance running keycloak and right now prod doesn't allow registration yet so the links will tell you registration isn't allowed. If you want to test it locally, you can open account.md and replace incore.ncsa.illinois.edu with incore-dev.ncsa.illinois.edu. The link should take you to the registration page on dev.

**Review by ywkim312 — APPROVED** [review]
I have deployed this PR to dev and tested in there and looked good. Approve

**Review by ylyangtw — APPROVED** [review]
Tested on dev. Looks good.

**Pull request merged**